pkg/api/datavolume/store.go

```diff
@@ -0,0 +1,44 @@
+package datavolume
+
+import (
+	"fmt"
+
+	"github.com/rancher/apiserver/pkg/apierror"
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/rancher/wrangler/pkg/schemas/validation"
+
+	cdiv1beta1 "github.com/rancher/harvester/pkg/generated/controllers/cdi.kubevirt.io/v1beta1"
+)
+
+const (
+	VMKindName = "VirtualMachine"
```

Comment:
We can use `kubevirt.io/client-go/api/v1alpha3.VirtualMachineGroupVersionKind.Kind`